src/shared/bus-util.c

```diff
@@ -128,8 +129,8 @@ int bus_event_loop_with_idle(
 
                 if (r == 0 && !exiting && idle) {
                         /* Inform the service manager that we are going down, so that it will queue all
-                         * further start requests, instead of assuming we are already running. */
-                        sd_notify(false, "STOPPING=1");
+                         * further start requests, instead of assuming we are still running. */
+                        sd_notify(false, NOTIFY_STOPPING);
```

Comment:
if you touch this, maybe also wrap the invocation in `(void)` since we knowingly ignore the return code here